### PR TITLE
Fix ws methods

### PR DESCRIPTION
**Pull request by KlonD90**
Shutdown and exit for ws

Exit now only close connection

### CODE REVIEW

File: tarantool-lsp/websocket-lib.lua [hunk 1]
```diff
@@ -21,7 +18,9 @@ local create_handler = function(options)
     log.info('Path: %s', path)
 
     if not initalized then
-        local ok, err = docs:init({ completions_dir = fio.pathjoin(path, 'completions') })
+        local ok, err = docs:init({
```
Comment (artur-barsegyan) [inline tarantool-lsp/websocket-lib.lua]: Why you have two separate pieces of init code?

Comment (artur-barsegyan) [reply tarantool-lsp/websocket-lib.lua]: I mean, do you have an ability to use default handlers from `tarantool-lsp/methods.lua`?

Comment (KlonD90) [reply tarantool-lsp/websocket-lib.lua]: For support use other completions

Comment (KlonD90) [reply tarantool-lsp/websocket-lib.lua]: But by default it will use default handlers

Comment (artur-barsegyan) [reply tarantool-lsp/websocket-lib.lua]: You can use the default handler with different options passed into the handler arguments

**Review by artur-barsegyan — COMMENTED** [review]
In general: changes are not affected the previous comments. Why you can't call `method_handlers(...)` with different options from WebSocket handler

**Review by artur-barsegyan — APPROVED** [review]

**Pull request merged**